ownCloud/FileLists/QueryFileListTableViewController.swift

```diff
@@ -470,6 +470,13 @@ class QueryFileListTableViewController: FileListTableViewController, SortBarDele
 			selectDeselectAllButtonItem?.target = self
 			selectDeselectAllButtonItem?.action = #selector(selectAllItems)
 		}
+		if selectedCount == 1 {
+			sortBar?.updateSelectedLabel(title: String(format: "%d Item selected".localized, selectedCount))
```

Comment:
@hosy Not really a complain, but a short side-note: we could consider using .stringdict files for plural localizations